components/motor/fake/motor.go

```diff
@@ -18,7 +19,16 @@ import (
 	"go.viam.com/rdk/resource"
 )
 
-var model = resource.DefaultModelFamily.WithModel("fake")
+var (
+	motorModel    = resource.DefaultModelFamily.WithModel("fake")
```

Comment:
prefer to keep it as mdoel unless there's a reason to change the variable name, most of the time this is two model names in the package, which isn't the case here
```suggestion
	model    = resource.DefaultModelFamily.WithModel("fake")
```